### DISCUSSION TITLE
Can I execute multiple SQL statements at once with C++ API

### DISCUSSION THREAD

**Discussion by lkordos** [duckdb/duckdb#12641] [Q&A]
The C++ API function Query, executed on a connection, takes as argument one SQL statement:
`con->Query(sql);`

Is it possible to execute multiple SQL statements with C++ API at once, or do I need to call each statement separately, using the Query API function?

**Comment by Tishj** [comment]
You can use multiple comma separated sql queries, provided as a string

**Reply by lkordos** [reply]
I tried it before, but it seems that only the last statement will be executed.

**Reply by Tishj** [reply]
They all get executed, only the last result is returned

**Reply by lkordos** [reply]
I see. Thanks.
